invokeai/frontend/web/src/features/system/components/SiteHeader.tsx

```diff
@@ -34,87 +33,58 @@ const SiteHeader = () => {
       <Spacer />
       <StatusIndicator />
 
-      <HotkeysModal>
-        <IAIIconButton
-          aria-label={t('common.hotkeysLabel')}
-          tooltip={t('common.hotkeysLabel')}
-          size="sm"
-          variant="link"
-          data-variant="link"
-          fontSize={20}
-          icon={<FaKeyboard />}
-        />
-      </HotkeysModal>
-
-      {isLocalizationEnabled && <LanguagePicker />}
-
-      {isBugLinkEnabled && (
-        <Link
-          isExternal
-          href="http://github.com/invoke-ai/InvokeAI/issues"
-          marginBottom="-0.25rem"
-        >
-          <IAIIconButton
-            aria-label={t('common.reportBugLabel')}
-            tooltip={t('common.reportBugLabel')}
-            variant="link"
-            data-variant="link"
-            fontSize={20}
-            size="sm"
-            icon={<FaBug />}
-          />
-        </Link>
-      )}
-
-      {isGithubLinkEnabled && (
-        <Link
-          isExternal
-          href="http://github.com/invoke-ai/InvokeAI"
-          marginBottom="-0.25rem"
-        >
-          <IAIIconButton
-            aria-label={t('common.githubLabel')}
-            tooltip={t('common.githubLabel')}
-            variant="link"
-            data-variant="link"
-            fontSize={20}
-            size="sm"
-            icon={<FaGithub />}
-          />
-        </Link>
-      )}
-
-      {isDiscordLinkEnabled && (
-        <Link
-          isExternal
-          href="https://discord.gg/ZmtBAhwWhy"
-          marginBottom="-0.25rem"
-        >
-          <IAIIconButton
-            aria-label={t('common.discordLabel')}
-            tooltip={t('common.discordLabel')}
-            variant="link"
-            data-variant="link"
-            fontSize={20}
-            size="sm"
-            icon={<FaDiscord />}
-          />
-        </Link>
-      )}
-
-      <ColorModeButton />
-
-      <SettingsModal>
-        <IAIIconButton
-          aria-label={t('common.settingsLabel')}
-          tooltip={t('common.settingsLabel')}
+      <Menu>
+        <MenuButton
+          as={IAIIconButton}
           variant="link"
-          data-variant="link"
-          fontSize={22}
-          size="sm"
-          icon={<MdSettings />}
+          aria-label={t('accessibility.menu')}
+          tooltip={t('accessibility.menu')}
+          icon={<FaBars />}
         />
-      </SettingsModal>
+        <MenuList>
+          <MenuGroup title={t('common.communityLabel')}>
+            {isGithubLinkEnabled && (
+              <MenuItem
+                as="a"
+                href={githubLink}
+                icon={<FaGithub fontSize={20} />}
```

Comment:
either no icons, or default icons size. the rest of the app's context menus use the default size, if you want to change them, please make that a follow up PR that changes them for the whole app. they need to be consistent